galaxy/tools/deps/mulled/mulled_search.py

```diff
@@ -4,6 +4,13 @@
 import json
 import sys
 import tempfile
+import urllib2
+import logging
+
+from conda.cli.python_api import run_command
```

Comment:
This should be conditionally imported like whoosh is below, this isn't available on PyPI but this should work with PyPI installs right? Even if you decide this functionality should require galaxy-lib be running in a Conda environment I'd like to see the file importable without errors under vanilla Python and just throw an error if one attempts to run it without the required dependencies.

Reply:
Okay, that makes sense.